sonar-plugin-api/src/main/java/org/sonar/api/web/ServletFilter.java

```diff
@@ -140,7 +140,7 @@ public static Builder builder() {
      */
     public static class Builder {
       private static final String WILDCARD_CHAR = "*";
-      private static final Collection<String> STATIC_RESOURCES = unmodifiableList(asList("/css/*", "/fonts/*", "/images/*", "/js/*", "/static/*",
+      private static final Collection<String> STATIC_RESOURCES = unmodifiableList(asList("*.css", "*.css.map", "*.ico", "*.png", "*.gif", "*.svg", "*.js", "*.js.map", "*.eot", "*.ttf", "*.woff", "/static/*",
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Split this 207 characters long line (which is greater than 180 authorized). ([squid:S00103](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3121&issues=AWILuujX54IFfP4XjVw1&open=AWILuujX54IFfP4XjVw1)

[ISSUE_KEY]: AWILuujX54IFfP4XjVw1